dev-packages/application-package/src/application-props.ts

```diff
@@ -45,11 +74,16 @@ export namespace ElectronFrontendApplicationConfig {
         readonly windowOptions?: BrowserWindowConstructorOptions;
 
         /**
-         * Whether or not to show an empty Electron window as early as possible.
+         * Whether or not to show an empty Electron main window as early as possible.
          *
-         * Defaults to `true`.
+         * Has no effect if `splashScreenOptions.content` is also given.
```

Comment:
good idea, that will clarify the functionality automatically from an adopter point of view.

Initially I thought we could support both, `showWindowEarly` and `splashScreenOptions` by showing the main window and the splash screen in case that `showWindowEarly` is true. However as it's true by default, almost all adopters which want to use the splash screen then would need to turn it off, which is why I did not go forward with it.